cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectBuildUtils.java

```diff
@@ -83,4 +83,17 @@ public static boolean buildSucceeded(Build build)   {
     public static boolean buildComplete(Build build)   {
         return buildSucceeded(build) || buildFailed(build);
     }
+
+    /**
+     * Get ContainerStatus of the Connect build container
+     *
+     * @param pod Pod which contains ContainerStatus of the Connect build
+     *
+     * @return ContainerStatus of the build container
+     */
+    public static ContainerState getConnectBuildContainerStatus(Pod pod)   {
+        ContainerStatus containerStatus = pod.getStatus().getContainerStatuses().stream()
+            .filter(conStatus -> conStatus.getName().equals(pod.getMetadata().getName())).findFirst().orElseGet(ContainerStatus::new);
```

Comment:
Do we have something better to use here than `pod.getMetadata().getName()` for container name? Seems a bit wierd to me.

Reply:
That would be better, yes. I wonder why the Pod name is the container name. But if we did it like that, it might not make sense to change it right now.